packages/button/button.scss

```diff
@@ -30,11 +30,17 @@ $hover-elevation-distance: -0.25rem;
     transition: transform $animation-timing;
     position: relative;
 
-    &:focus,
+    &:-moz-focusring {
+        // get rid of dotted border on Firefox
+        outline: none;
+    }
+
+    html:not([data-mousenavigation]) &:focus,
```

Comment:
Have you spoken with the designers about this? The focused-state does not have any scaling in Figma and this is a design change.

Reply:
Good point! Checked, and was OK 👍